test/ANNOTATIONS.yaml

```diff
@@ -769,14 +788,10 @@ prk-stencil.ml-perf:
     - Use nonblocking transactions for strided transfers (#6262)
   06/06/17:
     - Increase the PRK stencil problem size (#6380)
-  06/13/17:
-    - Optimize updateFluff (#6418)
 
 prk-stencil-time:
   05/11/17:
     - Move chpl_getPrivatizedClass() into chpl-privatization.h (#6212)
-  06/13/17:
-    - Optimize updateFluff (#6418)
```

Comment:
I messed up the first time around, they didn't actually do anything to the stencil as far as I could tell. I can put them back in though.